prql-compiler/tests/integration/snapshots/integration__tests__rdbms@invoice_totals.prql.snap

```diff
@@ -0,0 +1,360 @@
+---
+source: prql-compiler/tests/integration/main.rs
+expression: result_string
+input_file: prql-compiler/tests/integration/queries/invoice_totals.prql
+---
+2009-01,2009-01-01,1,2,1.98,2,
```

Comment:
Not on you at all, and def not necessary in this PR, but one thing we could do is adjust the queries so they produce more aggregated results — we probably get the same test coverage with more legible outputs...

Reply:
Sure.
And we have to remove the skips for certain dbs. Especially MsSQL is hardly tested.